Update operating-synergy.md (#164)
AvolitesLtd/TitanManual: docs/synergy/operating-synergy.md

codec as required by the Ai system. This is very handy when you are
handed a last-minute media file at the front of house position.

> As like as any other window the media browser window is opened through the
> \[Open Workspace Window\] menu.

Reviewer: ```suggestion
As with other windows, the media browser window is opened through the
\[Open Workspace Window\] menu, by pressing \<Open\>/\<View\>.
```

Reviewer 2: Suggest using the double press "View" method rather than the Workspace Windows menu as we are recommending that everywhere else in the manual.

Reviewer: Would you like me to make this change?

Author: yes please...

Reviewer 2: Yes I am not sure how to edit your edit
